docs/ENVIRONMENT.md

```diff
@@ -35,6 +35,9 @@ All tools:
   as `start` into no-ops.  If that's what's explicitly desired, you might
   consider setting `$SYSTEMD_OFFLINE=1`.
 
+* `$SYSTEMD_IN_CHROOT=0|1` — A more general form of `$SYSTEMD_IGNORE_CHROOT` that
+  overrides chroot detection if set.
```

Comment:
Please deprecate `SYSTEMD_IGNORE_CHROOT` then, i.e. remove from documentation and add a comment in code